index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="style.css">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name=" viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Skeleton</title>
+</head>
+<body>
+  <div id="mobile-background">
+    <div id="Status-bar"></div>
+    
+    <header>
+      <a href="#">
+        <h2>Inyambo</h2>  
```

Comment:
[OPTIONAL]
-  Good job adding the title in the header tag 👍🏼 . Kindly note that it is against the heading hierarchy to have the `<h2>` tag coming before the `<h1>`tag, the order of the heading hierarchy is `h1-h6`.

 I think it would be better if you wrap the title in the `<h1>` tag or just write the text in the anchor <a> tag to make your website semantic 😄